development/decision_process.rst

```diff
@@ -0,0 +1,182 @@
+
+In our RFC process, the opinions of those who will be doing the work (and fixing any problems if something goes wrong) are given more weight.
+In some cases, this may mean that the RFC issue's Assignee passes to someone else.
+The opinions of more senior people or people more experienced in the area should also be given more weight and may also result in the Assignee changing.
+
+The Assignee is responsible for determining when no serious objections remain.
+In particular, there is no need to call for a formal vote on the (refined) resolution.
+If no explicit objections have been raised within, typically, 72 hours for "ordinary" issues and 1 week for "major" issues, the Assignee should assume that there are none.
+This is known as "lazy consensus."
+
+Part of the consensus-building process is an agreement over the JIRA tickets that will implement the proposal.
+Use an 'Is triggered by' JIRA linkage for implementation tickets.
+
+When consensus is established, and a set of implementation tickets is created, the Assignee is responsible for marking the RFC as **Adopted** in JIRA.
+
+Be especially careful about not making irreversible changes in the "lazy consensus" time period unless you're absolutely certain there's a general agreement on the stated course of action.
+If you break something, be ready to fix it.
+Apply sound reasoning and good judgment about what may be acceptable and what might be not.
+Mistakes will happen; accept that occasionally you will be requested to revert an action for which you thought agreement existed, and learn from the experience.
+
+.. _decision-making-rfc-appeals:
+
+Appeals process
+---------------
+
+If you can't converge on a resolution to an RFC that has no serious objections but you still feel that something must be done, you may request that the DM PM and PS rule on it.
+In most non-trivial cases, they will, with the advice of the SA, impanel a group of experts to which they will delegate the right to make the decision, by voting if need be.
+
+.. _decision-making-rfc-implementation:
+
+RFC implementation
+------------------
+
+.. As you implement an RFC, remember to link your JIRA stories to the original RFC.
+.. This can be done through an "Is triggered by" or "Is related to" linkage.
+.. This helps the team track the status of an RFC's implementation.
+
+An RFC is implemented by completing all tickets that were created and agreed upon during the RFC discussion.
+
+Once the RFC is implemented, return to the RFC's JIRA issue page and click the "We Shipped It!" button.
+This changes the RFC's status from "Adopted" to "Implemented."
+
+.. _decision-making-rfc-tct:
+
+RFCs that affect change-controlled documents
+--------------------------------------------
+
+Some proposed resolutions may require changes to one or more of the baselined, change-controlled documents describing the Data Management system (those in DocuShare_ with an LDM- handle or marked as change-controlled in Confluence).
```

Comment:
And now we have documents (like this one) outside Confluence that need to be change-controlled.